src/app/House-data/house-data.component.html

```diff
@@ -0,0 +1,68 @@
+<!--   Header   -->
+<div class="section no-pad-bot" id="index-banner">
+  <div class="container">
+    <h5 class="header center green-text small">Binary classification with TensorFlow.js</h5>
+    <div class="row center">
+      <h6 class="header col s12 light">Train a model to predict a class.</h6>
+    </div>
+  </div>
+</div>
+
+<!--   Misc buttons   -->
+<div class="section no-pad-bot light-green lighten-4">
+  <div class="container">
+    <div class="row center">
+      <button id="toggle-button" class="waves-effect waves-light light-green btn-small" (click)="toggleVisor()">Toggle Visor</button>
+      <br/><br/>
+    </div>
+  </div>
+</div>
```

Comment:
Which **css** framework you are using in  `house-data.component.html`

Reply:
`Materialize` for now but will be redesigning this